api/v1beta1/nutanixmachine_types.go

```diff
@@ -92,6 +92,11 @@ type NutanixMachineSpec struct {
 	// +optional
 	BootstrapRef *corev1.ObjectReference `json:"bootstrapRef,omitempty"`
 
+	// bootstrapImageName is the image name of the bootstrap data.
+	// The image must be available from PC if configured.
+	// +optional
+	BootstrapImageName *string `json:"bootstrapImageName,omitempty"`
```

Comment:
IMO we should add a special field for bootstrap Image. We should instead have a field for additional disks. That is a more generic solution and can be used for mounting disks other than bootstrap disks for other use cases.